src/react/boosts/BoostTextComponents.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from "react";
+import { ModalHeading, ModalContent } from "@joincivil/components";
+
+export const WhyEthModalText: React.FunctionComponent = props => (
+  <>
+    <ModalHeading>
+      Why ETH?
+    </ModalHeading>
+    <ModalContent>
+      <p>100% of your ETH will go right into the newsroom’s wallet. This way, the Newsroom get’s your full support.</p>
```

Comment:
Typos in all 3 of these paragraphs, most of which are fixed in the designs https://consensys.invisionapp.com/d/main#/console/17414610/367963608/comments but I added a couple additional comments/suggestions to the text there so I'd update from there.